test/integration/sign-up-with-id-test.js

```diff
@@ -15,8 +16,10 @@ var options = {
 
 test('sign up with id', function (t) {
   store.clear()
-  t.plan(5)
+  t.plan(6)
 
+  // mocks
```

Comment:
Just pushed up the changes with more explicit comments. I was wondering if adding a account property to the state object in [here](https://github.com/hoodiehq/hoodie-account-client/pull/117/commits/dfbaaefca1dcddb4c7289788657936e1ff70b7f0) was fine?